src/chart/generateCategoricalChart.tsx

```diff
@@ -2078,19 +2087,51 @@ export const generateCategoricalChart = ({
       return result;
     };
 
+    renderActiveBar = ({
+      item,
+      childIndex,
+    }: {
+      item: GraphicalItem<ComponentProps<typeof Bar>>;
+      childIndex: number;
+    }) => {
+      const { key } = item.props;
+      const { activeBar, dataKey } = item.item.props;
+
+      /**
+       * Default to inheriting props from the parent bar
+       */
+      const inheritedProps = {
+        ...item.item.props,
+        ...item.props.data[childIndex],
+      };
+
+      const barProps = {
+        index: childIndex,
+        dataKey,
+        fill: getMainColorOfGraphicItem(item.item),
+        strokeWidth: 2,
+        stroke: '#fff',
```

Comment:
Nit: It would be nice to extract hardcoded constants named to the top of the file (we might even want to reuse constant across files as consistent defaults).